storybook/src/components/PageHeader/PageHeader.docs.mdx

```diff
@@ -56,7 +56,18 @@ The text of the menu button and the accessible navigation description can be cus
 
 <Canvas of={PageHeaderStories.WithCustomTexts} />
 
-### With a custom link component
+### With custom logo
```

Comment:
Maybe we should move this story below 'With custom link component'? That story is relevant for user within the City of Amsterdam, this story is only relevant for users outside it.